scripts/create_resources.sh

```diff
@@ -4,15 +4,17 @@ PROJECT=$1
 SERVER_NAME=$2
 COMPUTE_NAME=$3
 BUCKET=$4
+CIDR=$5
+GC_REGION=$6
 
 NETWORK=cloud-workflows
 SUBNET=cloud-workflows-default
 
 COMPUTE_ACCOUNT="$COMPUTE_NAME@$PROJECT.iam.gserviceaccount.com"
 SERVER_ACCOUNT="$SERVER_NAME@$PROJECT.iam.gserviceaccount.com"
 
-WASHU_CIDR="128.252.0.0/16"
-WASHU2_CIDR="65.254.96.0/19"
+# WASHU_CIDR="128.252.0.0/16"
+# WASHU2_CIDR="65.254.96.0/19"
```

Comment:
Agree with parameterizing this - would be good to make sure the docs include these ranges for easy reference.